### DISCUSSION TITLE
Standard edge-label prediction datasets

### DISCUSSION THREAD

**Discussion by LucaCappelletti94** [snap-stanford/ogb#433] [General]
Hi all,

I am looking for standard edge-label prediction datasets to evaluate a new model.

If I am not mistaken, this task is not yet covered by OGB.

The [BBOP](http://www.berkeleybop.org/) team has developed [several KGs and ontologies](https://kghub.org/) which may be used for such a task, as most of them have edge labels.

Currently, [800 different KGs versions are available from KGOBO and KGHUB, with 211 unique graphs](https://github.com/AnacletoLAB/grape/blob/main/tutorials/Ensmallen_Automatic_Graph_Retieval_KGOBO.ipynb). I hope that among these we could possibly find a few that are compatible with your data quality standards.

A knowledge graph I believe may be up to this task is [KGCOVID19](http://dx.doi.org/10.1016/j.patter.2020.100155), which you can readily retrieve by running (after installing grape with `pip install grape`):

```python
from grape.datasets.kghub import KGCOVID19

kg = KGCOVID19()
```

This will [download the edge list from kghub](https://kg-hub.berkeleybop.io/kg-covid-19/index.html). I suggest using the aforementioned grape library so you will get a [comprehensive graph report](https://github.com/AnacletoLAB/grape/blob/main/tutorials/Create%20extensive%20knowledge%20graph%20reports%20using%20GRAPE.ipynb) to better evaluate whether this (or other) available KGs may be fit as a standard edge-label prediction benchmark dataset.

I hope it will be possible to readily agree on some standard dataset for edge-label prediction.

I haven't found a discord server for OGB, but if you'd like to quickly talk about this I am available on [GRAPE's discord server](https://discord.gg/Nda2cqYvTN).

Best,
Luca

**Comment by weihua916** [comment]
Hi Luca,

Thanks for reaching out! You are right that we do not have edge-label prediction datasets in OGB, so we welcome dataset contribution here! [Here](https://docs.google.com/document/d/e/2PACX-1vS1hBTYLONRwAU9UxK42USTuRKrt_Yk4H0EhpLvJC_eOrGxbJUtrzDqlIStAFpnwZt2N28B3MuSxgqj/pub) is the instruction how you can contribute some of the datasets! Let us know if you have any further questions.

Best,
Weihua

**Reply by LucaCappelletti94** [reply]
I'll get started, thanks for proving that link.
